R/undesirable_function_linter.R

```diff
@@ -4,7 +4,9 @@
 #' [base::sapply()] and suggest an alternative.
 #'
 #' @param fun Named character vector, where the names are the names of the undesirable functions, and the values are
-#'   the text for the alternative function to use (or `NA`).
+#'   the text for the alternative function to use (or `NA`). Defaults to
```

Comment:
This is no longer correct. The values are additional hints, since they're added to the lint message as-is.